content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,92 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
+
+Although the experiments as the temporary, they can be shared with
+`dvc exp push` and retrieved from other Git repositories with `dvc exp pull`. In
+this section we'll see how to bring them to the standard Git workflow with
+`dvc exp apply` and `dvc exp branch`.
+
+## Bring Experiment results to your workspace
+
+Typically, `dvc exp run` leaves the experiment
+results in your workspace for convenience. However, you may have run
+multiple experiments and wish to go back to a specific one. In this case,
+you can restore a previous experiment's results with `dvc exp apply`.
+Let's see an example:
+
+```dvc
+$ dvc exp show --include-params=my_param
+```
+
+```dvctable
+┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┓
+┃ Experiment            ┃ Created      ┃     auc ┃ my_param   ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━┩
+│ workspace             │ -            │ 0.61314 │ 3          │
+│ experiments-tag       │ Oct 19, 2020 │ 0.61314 │ 3          │
+│ ├── exp-e6c97         │ Oct 20, 2020 │ 0.69830 │ 2          │
+│ └── exp-1df77         │ Oct 22, 2020 │ 0.51676 │ 1          │
+└───────────────────────┴──────────────┴─────────┴────────────┘
+```
+
+Here, the results found in the workspace in the respective row. When you want to
+bring another experiment from the listed experiments, you can do so with:
+
+```dvc
+$ dvc exp apply exp-e6c97
+Changes for experiment 'exp-e6c97' have been applied...
+```
+
+Now, if you list the experiments again with `dvc exp show`, you'll see that the
+workspace contains the experiment with `my_param` of `2` and corresponding `auc` value.
```

Comment:
> the workspace contains the experiment with with `my_param` of `2`

- We should reiterate the ID I think (`exp-e6c97`).
- And also, what does it mean that the workspace "contain an experiment"? We should be explicit: both the setup (dependencies, params, etc.) including data and code, and the results (artifacts or ML model in this case? and metrics).

Reply:
Not all of them are restored from DVC cache, textual content is restored from Git objects. I'll revise this but I think it's better not to mention DVC cache explicitly here.

Reply:
`dvc exp show` doesn't show an experiment id when it's in the workspace. It just says `workspace.` I didn't put the experiment ID there because of this. `exp-e6c97` becomes "the experiment in the `workspace`" after `dvc exp apply`.

Reply:
> dvc exp show doesn't show an experiment id when it's in the workspace

Yeah I meant in the text e.g. "you'll see that the workspace contains the results of experiment `exp-e6c97`..."

> Not all of them are restored from DVC cache

OK you can just end the sentence at "restored." Or alternatively just say "contains the results of" like I suggested above and don't go into details.

But I see this whole section was removed? Do you think it's not needed? Up to you then

Reply:
I didn't remove the section, changed the order of sections. This is done in dd889ca.

Reply:
Looks good!